api/pkg/cli/cmd/install/install_test.go

```diff
@@ -96,7 +96,7 @@ func TestInstall_NewResource(t *testing.T) {
 
 	err := opts.run()
 	assert.NoError(t, err)
-	assert.Equal(t, "Task foo(0.3) installed in hub namespace\n", buf.String())
+	assert.Equal(t, "WARN: tekton pipelines version unknown, this resource is compatible with pipelines min version v0.12\nTask foo(0.3) installed in hub namespace\n", buf.String())
```

Comment:
same. if the task is getting installed then should we warn?

Reply:
samee as said above. If Tekton Pipelines is installed on the cluster but in another namespace or we don't have sufficient permissions to view the version of the Tekton Pipelines installed then we can give the above warning.